### PR TITLE
Feature 11784 cpus maintenance

### PR DESCRIPTION
**Pull request by Alfro**

### CODE REVIEW

File: src/Pumukit/EncoderBundle/Services/JobService.php [hunk 1]
```diff
@@ -440,6 +445,13 @@ public function execute(Job $job)
 
             $multimediaObject = $this->getMultimediaObject($job);  //Necesary to refresh the document
             $this->propService->errorJob($multimediaObject, $job);
+            // If the transco is disconnected or there is an authentication issue, we don't want to send more petitions to this transco.
+            if( strpos($e->getMessage(), 'HTTP 401')
```
Comment (rubenrua) [inline src/Pumukit/EncoderBundle/Services/JobService.php]: ?!?!?!?

Comment (rubenrua) [inline src/Pumukit/EncoderBundle/Services/JobService.php]: See commit creating custom exception: `ExecutorException`: b9a3ef7c565eb026870542ce9dfa8fc1aecc5568

**Comment by Alfro** [comment]
It should be alright now.

File: src/Pumukit/EncoderBundle/Controller/CPUsController.php [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    public function switchMaintenanceAction(Request $request, $activateMaintenance, $cpuName)
+    {
+        $cpuService = $this->get('pumukitencoder.cpu');
+        $cpu = $cpuService->getCpuByName($cpuName);
+        if(!$cpu) {
+            throw $this->createNotFoundException("The CPU with the name $cpuName does not exist");
+        }
+        switch($activateMaintenance) {
+        case 'activate':
+            dump('activate');
```
Comment (rubenrua) [inline src/Pumukit/EncoderBundle/Controller/CPUsController.php]: dump!!!!!!

File: src/Pumukit/EncoderBundle/Resources/views/Info/cpus.html.twig [hunk 3]
```diff
@@ -1,19 +1,46 @@
...
-      <p class="list-group-item-text">{{ cpu.description|default('No description') }} (type: {{ cpu.type }})</p>
-      <small>{{ cpu.host }}</small>
-    </div>
-  </div>
+    <script>
+     $('#maintenance-{{name}}-toggle').click(function(){
+         var url = '';
+         var activateUrl = '{{ path('pumukit_encoder_cpu_maintenance_switch', { 'activateMaintenance': 'activate' , 'cpuName': name} ) }}';
+         var deactivateUrl = '{{ path('pumukit_encoder_cpu_maintenance_switch', { 'activateMaintenance': 'deactivate' , 'cpuName': name} ) }}';
+         console.log(this.checked);
```
Comment (rubenrua) [inline src/Pumukit/EncoderBundle/Resources/views/Info/cpus.html.twig]: log!!!

**Comment by Alfro** [comment]
@rubenrua Hopefully this #229 will help preventing these kind of issues in the future :)

**Pull request merged**